### DISCUSSION TITLE
Install: error installing firedrake on windows wsl with ubuntu

### DISCUSSION THREAD

**Discussion by Martin2523** [firedrakeproject/firedrake#3494] [Firedrake support]
Hi,
I am trying to install firedrake on my wsl with ubuntu and I keep getting the same error.  After `python3 firedrake-install` this is what I get:
Running firedrake-install
Traceback (most recent call last):
  File "/mnt/c/Users/marti/documents/codes/firedrake-install", line 575, in <module>
    fortran = sniff_compiler(args.mpif90 or "gfortran")
  File "/mnt/c/Users/marti/documents/codes/firedrake-install", line 514, in sniff_compiler
    output = subprocess.run(
  File "/usr/lib/python3.10/subprocess.py", line 503, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.10/subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.10/subprocess.py", line 1863, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'gfortran'

can someone help me? I've also tried installing in my linux home directory instead of `/mnt/c` as the guide says but the error is exactly the same. 
this is the log file 
[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/14905305/firedrake-install.log)

Thank you!

**Comment by dham** [comment]
I think this is a bug in the install script where we're checking which `gfortran` you have before we actually install it. Can you try installing gfortran manually and then running the install:
```console
sudo apt install gfortran
```

**Reply by Martin2523** [reply]
Yes, it worked after installing gfortran manually.

Thank you very much!
